docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -124,6 +124,7 @@ Basic layers:
 
 Basic layers:
 
+* _batch-jberet_: Tech preview support for Jakarta Batch.
```

Comment:
@chengfang Just for your info, we will remove the "Tech preview" from this community documentation on this PR: https://github.com/wildfly/wildfly/pull/13235

It should not be here in the documentation.